converter/internal/promtailconvert/promtailconvert.go

```diff
@@ -128,7 +128,6 @@ func appendScrapeConfig(
 	gctx *build.GlobalContext,
 ) {
 	//TODO(thampiotr): need to support/warn about the following fields:
```

Comment:
does this comment go away now that you've added the diag plus the TODO with the linked issue?

Reply:
Yes! thanks for noticing